pandas/core/generic.py

```diff
@@ -56,7 +56,7 @@
 # able to share
 _shared_docs = dict()
 _shared_doc_kwargs = dict(
-    axes='keywords for axes', klass='NDFrame',
+    axes='keywords for axes', klass='DataFrame',
```

Comment:
Inside this file (generic.py), I would replace all occurences of 'NDFrame' with `"Series/DataFrame"`, since those docstrings might be shared by both Series and DataFrame

Reply:
Will do!

Reply:
@jorisvandenbossche I have fixed the issue. How do you think?